defs/go.bzl

```diff
@@ -57,13 +60,15 @@ def _go_genrule_impl(ctx):
         # Set GOPATH and GOROOT to absolute paths so that commands can chdir without issue
         "export GOPATH=" + ctx.configuration.host_path_separator.join(["$$GO_GENRULE_EXECROOT/" + p for p in go_paths]),
         "export GOROOT=$$GO_GENRULE_EXECROOT/" + go.root,
+        "export PATH=$$GO_GENRULE_EXECROOT/" + go.root + "/bin:$$PATH",
```

Comment:
ah, I see what you fixed here (`go.root` is relative) can you maybe also remove `paths` below?

Reply:
oops, will do